UNRELEASED.md

```diff
@@ -2,6 +2,17 @@
 
 > Place your changes below this line.
 
+**Added:**
+
+- bpk-component-icon:
+- bpk-svgs:
+- bpk-mixins:
+  - Updated family icon
+  - Added filter icon
+  - Added world-amer icon
```

Comment:
Can we put the icon names in backticks? Also the names include two dahes e.g. `world--amer` not `world-amer`

Reply:
Good catch!